src/config/packets.h

```diff
@@ -9,16 +9,12 @@
  **/
 
 #ifndef PACKETVER
+	///Do not edit this line. To set your client version, please do this instead.
+	///In Linux: run the following command: ./configure --enable-packetver=YYYYMMDD
+	///In Windows: add this line in your src\custom\defines_pre.h file: #define PACKETVER YYYYMMDD
 	#define PACKETVER 20151104
 #endif
 
-#ifndef PACKETVER_RE
-	/// From this point on only kRO RE clients are supported
-	#if PACKETVER > 20151104
-		#define PACKETVER_RE
-	#endif
-#endif
-
```

Comment:
Is there a reason this was moved down?

Reply:
some people get confused by that `#define PACKETVER 20151104` is the same of `#if PACKETVER > 20151104` so they edit both
while we can't control the server owner option to edit the version in here when he edit both to put his version while he use RagexeRE +2016 he will get the error in the screenshot above
moving it down will decrease the error rate by the servers owner
however if people start to use the other methods to set there version , no one would get the error and get confused BY `#define PACKETVER` AND `#if PACKETVER`